app-misc/broot/broot-1.44.7.ebuild

```
@@ -0,0 +1,460 @@
+snafu@0.7.5
+snafu-derive@0.7.5
+splitty@1.0.2
+stable_deref_trait@1.2.0
+str-buf@1.0.6
+strict@0.1.4
+strict@0.2.0
+strict-num@0.1.1
+strsim@0.11.1
+svg@0.16.0
+svgtypes@0.15.3
+syn@1.0.109
+syn@2.0.98
+synstructure@0.13.1
+syntect-no-panic@6.0.0
+system-deps@6.2.2
+target-lexicon@0.12.16
+tempfile@3.16.0
+termimad@0.31.2
+terminal-clipboard@0.4.1
+terminal-light@1.7.0
+termux-clipboard@0.1.0
+thiserror@1.0.69
+thiserror-impl@1.0.69
+tiff@0.9.1
+time@0.3.37
+time-core@0.1.2
+time-macros@0.2.19
+tiny-skia@0.11.4
+tiny-skia-path@0.11.4
+tinystr@0.7.6
+tinyvec@1.8.1
+tinyvec_macros@0.1.1
+toml@0.8.20
+toml_datetime@0.6.8
+toml_edit@0.22.24
+trash@3.3.1
+ttf-parser@0.24.1
+umask@2.1.0
+unicode-bidi@0.3.18
+unicode-bidi-mirroring@0.3.0
+unicode-ccc@0.3.0
+unicode-ident@1.0.16
+unicode-normalization@0.1.24
+unicode-properties@0.1.3
+unicode-script@0.5.7
+unicode-vo@0.1.0
+unicode-width@0.1.14
+url@2.5.4
+usvg@0.44.0
+utf16_iter@1.0.5
+utf8_iter@1.0.4
+utf8parse@0.2.2
+uzers@0.12.1
+v_frame@0.3.8
+vcpkg@0.2.15
+version-compare@0.2.0
+version_check@0.9.5
+walkdir@2.5.0
+wasi@0.11.0+wasi-snapshot-preview1
+wasi@0.13.3+wasi-0.2.2
+wasm-bindgen@0.2.100
+wasm-bindgen-backend@0.2.100
+wasm-bindgen-macro@0.2.100
+wasm-bindgen-macro-support@0.2.100
+wasm-bindgen-shared@0.2.100
+weezl@0.1.8
+which@4.4.2
+winapi@0.3.9
+winapi-i686-pc-windows-gnu@0.4.0
+winapi-util@0.1.9
+winapi-wsapoll@0.1.2
+winapi-x86_64-pc-windows-gnu@0.4.0
+windows@0.44.0
+windows-core@0.52.0
+windows-sys@0.48.0
+windows-sys@0.52.0
+windows-sys@0.59.0
+windows-targets@0.42.2
+windows-targets@0.48.5
+windows-targets@0.52.6
+windows_aarch64_gnullvm@0.42.2
+windows_aarch64_gnullvm@0.48.5
+windows_aarch64_gnullvm@0.52.6
+windows_aarch64_msvc@0.42.2
+windows_aarch64_msvc@0.48.5
+windows_aarch64_msvc@0.52.6
+windows_i686_gnu@0.42.2
+windows_i686_gnu@0.48.5
+windows_i686_gnu@0.52.6
+windows_i686_gnullvm@0.52.6
+windows_i686_msvc@0.42.2
+windows_i686_msvc@0.48.5
+windows_i686_msvc@0.52.6
+windows_x86_64_gnu@0.42.2
+windows_x86_64_gnu@0.48.5
+windows_x86_64_gnu@0.52.6
+windows_x86_64_gnullvm@0.42.2
+windows_x86_64_gnullvm@0.48.5
+windows_x86_64_gnullvm@0.52.6
+windows_x86_64_msvc@0.42.2
+windows_x86_64_msvc@0.48.5
+windows_x86_64_msvc@0.52.6
+winnow@0.7.2
+wit-bindgen-rt@0.33.0
+write16@1.0.0
+writeable@0.5.5
+x11-clipboard@0.8.1
+x11rb@0.12.0
+x11rb-protocol@0.12.0
+xmlwriter@0.1.0
+xterm-query@0.5.0
+yaml-rust@0.4.5
+yoke@0.7.5
+yoke-derive@0.7.5
+zerocopy@0.7.35
+zerocopy-derive@0.7.35
+zerofrom@0.1.5
+zerofrom-derive@0.1.5
+zerovec@0.10.4
+zerovec-derive@0.10.3
+zune-core@0.4.12
+zune-inflate@0.2.54
+zune-jpeg@0.4.14
+"
+
+RUST_MIN_VER="1.79.0"
+inherit cargo shell-completion
+
+DESCRIPTION="A new way to see and navigate directory trees"
+HOMEPAGE="https://dystroy.org/broot/ https://github.com/Canop/broot"
+SRC_URI="https://github.com/Canop/broot/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	${CARGO_CRATE_URIS}"
+#SRC_URI+=" https://dev.gentoo.org/~arthurzam/distfiles/app-misc/${PN}/${P}-crates.tar.xz"
```

I have some good news. We now have a nice infra to automatically create for you tarballs using Gentoo controlled CI (might have delay of ~1h).

```suggestion
SRC_URI+=" https://github.com/gentoo-crate-dist/broot/releases/download/v${PV}/broot-v${PV}-crates.tar.xz"
```

So next time, use `pycargoebuild --no-write-crate-tarball` and I hope it would just work :)

---

Neat, thanks! I pushed a new version of the ebuild using the auto-built tarball.